src/core/modules/Planet.hpp

```diff
@@ -222,6 +222,7 @@ class Planet : public StelObject
 	virtual double getAngularSize(const StelCore* core) const Q_DECL_OVERRIDE;
 	virtual bool hasAtmosphere(void) {return atmosphere;}
 	virtual bool hasHalo(void) {return halo;}
+	bool hasValidPositionalData();
```

Comment:
Honestly, we need better documentation! The name is now better, but this should explain to other developers (or ourselves, in 4 months from now) why we need it, or what this test can be used for. E.g.

//! returns whether planet positions are valid and useful for the current simulation time. E.g. outdated orbital elements for Kepler orbits (beyond their orbit_good .ini file entries) may lead to invalid positions which should better not be used.

Reply:
OK, you are right. I'm added notes also...